### PR TITLE
fix(skate-style): Include landmarks tagged as nodes too

### PR DESCRIPTION
**Pull request by KaylaBrady**
ticket: https://app.asana.com/0/1148853526253437/1204073120343847/f
# Notes
## Separate skate-style `project.mml` file
I wanted to modify the existing `area_label` layer query. Since that is used by the default styles too, I opted to instead make a copy of `project.mml` specifically for `skate-style`.
## Changes to `area_label` layer
The `area_label` layer previously included `amenities`. I opted to remove `amenities` from this layer and create a new `landmarks` layer including only amenities for 2 reasons:
1. We want to include node amenities (which don't have area), so `area_label` seemed like a misnomer
2. hospitals & schools are the only amenities that were visibly labelled as part of `area_label`. It seemed easier to move these two into `landmarks` rather than add more types of amenities to the existing layer with more exceptions for zoom level behavior. This did result in some duplication of styles for 10-16 between `area_label` and `landmarks` though, and I'd be open to suggestions for other ways to handle!

### Removing invisible labels
I added filtering for `area_label` to only create labels for the 4 types of labels that had existing styling to be visible (3 greenspace, 1 water).
I also removed `highways` from the `area_label` layer since there are no highways labelled as part of this layer.

## Deployment considerations
Since browsers cache tiles, this may create some strangeness where new landmark markers are added that cross tile edges. We'll want to check on that in the dev environment before deploying to prod, and possibly try to alter the caching behavior.

### CODE REVIEW

**Comment by KaylaBrady** [comment]
Oops looked like I never saved the updated description 🤦‍♀️ 
Some more context incoming!

File: etc/skate-style/labels.mss [hunk 1]
```diff
@@ -320,109 +320,176 @@
...
+      [zoom=15][area>1600000],
+      [zoom=16][area>80000],
+      [zoom=17][area>20000],
+      [zoom=18][area>5000] {
+        text-name: "[name]";
+        text-size: 13;
+        text-wrap-width: 60;
+        text-character-spacing: 1;
+        text-halo-radius: 2;
+      text-clip: false;
```
Comment (lemald) [inline etc/skate-style/labels.mss]: **nitpick:** indentation

**Review by lemald — APPROVED** [review]
One super minor formatting thing but otherwise looks good to me!

**Review by firestack — APPROVED** [review]
Agree on the formatting suggestion above, but LGTM!

**Pull request merged**